app/controllers/likes_controller.rb

```diff
@@ -1,3 +1,21 @@
 class LikesController < ApplicationController
-  def create; end
+  # The function creates a like for a post and updates the post's like counter.
+  def create
+    @post = Post.find(params[:id])
+    @like = current_user.likes.build(like_params.merge(posts_id: @post.id))
+    if @like.save
+      @like.update_post_likes_counter
+      redirect_to user_post_path(@post), notice: 'Post liked successfully.'
+    else
+      render :show
+    end
+  end
```

Comment:
- [ ] Kindly make sure the user can be able to like the post successfully because at this moment when we try to like we get an error instead. Kindly fix that issue.

  ### ERROR DISPLAYED
![image](https://github.com/Benawi/Blog-App/assets/87197412/420f05b8-c8e6-4736-a5ce-26f1c29bea9d)